src/Agent.Worker/Telemetry/TelemetryCommandExtension.cs

```diff
@@ -82,13 +82,20 @@ private void ProcessPublishTelemetryCommand(IExecutionContext context, Dictionar
                 throw new ArgumentException(StringUtil.Loc("TelemetryCommandDataError", data, ex.Message));
             }
 
-            var ciService = HostContext.GetService<ICustomerIntelligenceServer>();
-            var vssConnection = WorkerUtilities.GetVssConnection(context);
-            ciService.Initialize(vssConnection);
+            try
```

Comment:
i think you might want to make the entire `ProcessPublishTelemetryCommand` best effort?
change any exception throw to be warning within this method

Reply:
I thought about that. but the area and feature other things are valid exception and best to be thrown. only calling http function should be ignore incase of error

Reply:
i am fine with this.